about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Contributing datasets
+nav_order: 1
+parent: About Data Commons
+---
+
+# Life of a Dataset
+
+This tutorial walks through the process of structuring and inserting data into the Data Commons graph.
+
+## Background
+
+Importing data into Data Commons boils down to taking source data and converting it into a format that can be consumed into the Data Commons graph. This section presents an example that does not require statistical data.
```

Comment:
I'm not a fan of the phrase "boils down" here -- it seems to colloquial. Can we use more formal phrasing here?

Reply:
Removed!